code-challenges.test.js

```diff
@@ -34,16 +72,79 @@ const nums2 = [109, 5, 9, 67, 8, 24]
 // Expected output: [5, 109]
 
 
-// b) Create the function that makes the test pass.
+describe('minMax', () => {
+    const nums1 = [3, 56, 90, -8, 0, 23, 6]
+    const nums2 = [109, 5, 9, 67, 8, 24]
+    it("takes an array of numbers and returns an array of the minimum and maximum numbers in that order.",() => {
+    expect(minMax(nums1)).toEqual([-8, 90])
+    expect(minMax(nums2)).toEqual([5, 109])
+    })
+})
+// Output:
+// FAIL  ./code-challenges.test.js
+// minMax
+//   ✕ takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
+// ● minMax › takes an array of numbers and returns an array of the minimum and maximum numbers in that order.
+//   ReferenceError: minMax is not defined
 
+// b) Create the function that makes the test pass.
+//Psuedocode
+// Create a function called minMax
+// takes in an array of numbers (array) parameter
+// use Math.min/max to get the values needed
+// return chosen values in new array
+
+const minMax = (array) => {
+    const smallNum = Math.min(...array);
+    const highNum = Math.max(...array);
+    return [smallNum, highNum]
+}
+
+//   console.log(minMax(nums1));
+//   console.log(minMax(nums2))
+// [ -8, 90 ]
+// [ 5, 109 ]
```

Comment:
This looks good! The fact that you have console.logs below suggests to me you are not relying on jest to see the output of your functions.